### PR TITLE
Alpha version release

### PR DESCRIPTION
**Pull request by eistrati**

### CODE REVIEW

File: src/AtmClient.php [hunk 1]
```diff
@@ -176,4 +188,146 @@ public function createLockedContent($content_id, $property_id, $text) {
...
+          'ConfigDefaults' => [
+            'revenueMethod' => $config['revenue_model'],
+            'content' => [
+              'offsetType' => $config['content_preview_type'],
+              'offset' => $config['content_preview'],
+              'lock' => $config['locking_algorithm'],
+            ],
+            'styles' => ['main' => $config['styles']],
+            'ads' => [
+              'relatedVideoCb' => "function(onReady) { onReady({ 'iPad,iPhone,Apple': ['https://www.youtube.com/watch?v=OzQh6wDb2oE&t=23s'] }) }",
```
Comment (AlexanderC) [inline src/AtmClient.php:277]: The `onReady` call should look like this: `onReady('https://www.youtube.com/watch?v=OzQh6wDb2oE&t=23s')`

File: src/AtmClient.php [hunk 2]
```diff
@@ -176,4 +188,146 @@ public function createLockedContent($content_id, $property_id, $text) {
...
+   *   Load template by name or null to load all templates.
+   *
+   * @return string|array|bool
+   *   Template markup, an array of all templates markup or FALSE on fail.
+   */
+  public function templatesLoad($name = NULL) {
+    $client = new Client();
+
+    try {
+      $request = $client->get('https://demo.adtechmedia.io/atm-admin/atm-build/templates.json');
```
Comment (AlexanderC) [inline src/AtmClient.php:311]: Please move the endpoint to config

File: src/Form/AdTechMediaConfigForm.php [hunk 3]
```diff
@@ -56,216 +56,260 @@ public function getEditableConfigNames() {
...
+      ],
+      '#limit_validation_errors' => [],
+    ];
 
-    $form['general']['country'] = array(
+    $form['general']['country'] = [
       '#type' => 'select',
       '#title' => $this->t('Country'),
-      '#options' => array(
+      '#options' => [
```
Comment (AlexanderC) [inline src/Form/AdTechMediaConfigForm.php:126]: Please use the `/atm-admin/property/supported-countries` endpoint to fetch such data from

File: src/Form/AdTechMediaConfigForm.php [hunk 4]
```diff
@@ -56,216 +56,260 @@ public function getEditableConfigNames() {
...
       '#title_display' => 'after',
-    );
+    ];
 
-    $form['content']['content_currency'] = array(
+    $form['content']['content_currency'] = [
       '#type' => 'select',
       '#title' => $this->t('Content currency'),
-      '#options' => array(
+      '#options' => [
```
Comment (AlexanderC) [inline src/Form/AdTechMediaConfigForm.php:178]: Please set it up depending on chosen country (@see `/atm-admin/property/supported-countries`)

**Review by AlexanderC — CHANGES_REQUESTED** [review]
Please check out and fix the issues pointed to.

**Comment by eistrati** [comment]
Please fix the problems pointed out by @AlexanderC

**Pull request closed without merging**